lib/config/v2/busway.ex

```diff
@@ -11,24 +11,27 @@ defmodule ScreensConfig.V2.Busway do
   typically split into multiple sections.
   """
 
-  alias ScreensConfig.V2.{Departures, EvergreenContentItem}
+  alias ScreensConfig.V2.{Audio, Departures, EvergreenContentItem}
   alias ScreensConfig.V2.Header.CurrentStopName
 
   @type t :: %__MODULE__{
           departures: Departures.t(),
           evergreen_content: list(EvergreenContentItem.t()),
-          header: CurrentStopName.t()
+          header: CurrentStopName.t(),
+          audio: Audio.t()
```

Comment:
In each case where this field is added, it's in a list that's currently alphabetized. Let's keep it that way?

Reply:
Happy to do that, but I think that might be coincidence. Bus shelter, for example, has it listed last as well. No harm in making the change though.